### ISSUE TITLE
Android App Link wakes up the app and fails

### ISSUE THREAD

**Issue by youxia-mrhan** [flutter/flutter#157146]
My project business needs to use App Link to wake up the App:

1. According to the official document, cooperate with Android Studio to complete the configuration, including adding `assetlinks.json` to the server;
https://developer.android.com/studio/write/app-link-indexing.html

2. After completing the configuration, use Android Studio to test, the server can access the file normally, https://xxx/.well-known/assetlinks.json, and it is consistent with the App package name and sha256, but it finally stops at the abnormal network connection.
<img width="1650" alt="截屏2024-10-18 12 41 27" src="https://github.com/user-attachments/assets/20ce884d-3d66-4a4d-b98c-6f2da318b4e7">

Question:

1. Is there a problem with my configuration?

2. Can Android App Link be used normally in mainland China?

3. Network problem? I turned on or off the network proxy, but it didn't solve the problem;

4. The server ports 80 and 443 are all open. In addition to adding `assetlinks.json`, do I need other configurations?

In other words, what additional configurations are required for servers in mainland China, such as installing a network proxy on the server;

**Comment by darshankawar** [comment]
@youxia-mrhan 
I don't think this issue belongs here. Maybe you can try to reach out in Google's android issue tracker for further resolution:

https://issuetracker.google.com/issues/new?component=190923&template=841312&pli=1

Check below links for further reference.

https://flutter-ko.dev/community/china

https://developer.android.com/studio/write/app-link-indexing.html#testindent

Closing for now, as this doesn't seem specific to Flutter. If you disagree, write in comments and I'll reopen it.
